internal/tools/go.mod

```diff
@@ -7,12 +7,14 @@ require (
 	github.com/frapposelli/wwhrd v0.4.0
 	github.com/fzipp/gocyclo v0.3.1
 	github.com/go-enry/go-license-detector/v4 v4.2.0
+	github.com/golang/mock v1.5.0
 	github.com/golangci/golangci-lint v1.40.1
 	github.com/gordonklaus/ineffassign v0.0.0-20210225214923-2e10b2664254
 	github.com/goware/modvendor v0.3.0
 	github.com/mgechev/revive v1.0.6
+	github.com/stormcat24/protodep v0.0.0-20210106175019-c95189bc461b
 	golang.org/x/mobile v0.0.0-20201217150744-e6ae53a27f4f
-	golang.org/x/perf v0.0.0-20200918155509-d949658356f9
+	golang.org/x/perf v0.0.0-20210220033136-40a54f11e909
```

Comment:
For future reference, why do we bump this?

Reply:
I think I just `go mod tidy` after removing the stuff, I didn't bump that specifically.